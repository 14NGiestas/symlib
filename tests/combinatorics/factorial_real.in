           3
           5
           4
           6
           6
           5
           2
           6
           2
           7
           3
           3
           4
           5
           5
