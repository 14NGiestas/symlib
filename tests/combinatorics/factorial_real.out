   6.0000000
 120.0000000
  24.0000000
 720.0000000
 720.0000000
 120.0000000
   2.0000000
 720.0000000
   2.0000000
5040.0000000
   6.0000000
   6.0000000
  24.0000000
 120.0000000
 120.0000000
